workflow/scripts/featurecounts_deseq2.R

```diff
@@ -317,11 +331,12 @@ if (file.exists(ResultsFile) == FALSE) {
             cat("\n",file=LogFile,append=TRUE,sep="")
         # AVI: creates required output files with message
         } else {
-            stop("More than 2 samples treated with the same antibody are needed to calculate the FDR & LOGFC.")
+            warning("More than 2 samples treated with the same antibody are needed to calculate the FDR & LOGFC.")
```

Comment:
The Snakemake rule will fail due to non-existant output, anyways, right? So we could also simply leave this as a hard stop here, so that the following code isn't executed in vain (as output gets deleted upon failure, anyways...)?

Reply:
No, the files are created empty or with their header only. The workflow would not stop. Since this is executed for each antibody and there can also be groups during a run which contain too few samples for the FDR analysis, but in contrast there are enough samples in other groups. I think there should not be a hard termination here. Otherwise, this would limit the design of the experiments and some experimental concepts would not be executable with our workflow. In Nextflow, the workflow is not stopped at this point, not even a warning is generated (https://github.com/nf-core/chipseq/blob/6924b669422215f9021144b251e83fc9929be1fe/bin/featurecounts_deseq2.r#L252). What do you think about this?

Reply:
Ah, yes. We discussed this previously. Sorry, I had forgotten. And yes, this makes absolute sense, then!